keyboards/ploopyco/ploopyco.c

```diff
@@ -16,118 +16,139 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-#include "mouse.h"
+#include "ploopyco.h"
+#include "analog.h"
+#include "opt_encoder.h"
 
-#ifndef OPT_DEBOUNCE
-#    define OPT_DEBOUNCE 5  // (ms) 			Time between scroll events
+// for legacy support
+#if defined(OPT_DEBOUNCE) && !defined(PLOOPY_SCROLL_DEBOUNCE)
+#    define PLOOPY_SCROLL_DEBOUNCE OPT_DEBOUNCE
 #endif
-#ifndef SCROLL_BUTT_DEBOUNCE
-#    define SCROLL_BUTT_DEBOUNCE 100  // (ms) 			Time between scroll events
+#if defined(SCROLL_BUTT_DEBOUNCE) && !defined(PLOOPY_SCROLL_BUTTON_DEBOUNCE)
+#    define PLOOPY_SCROLL_BUTTON_DEBOUNCE SCROLL_BUTT_DEBOUNCE
 #endif
-#ifndef OPT_THRES
-#    define OPT_THRES 150  // (0-1024) 	Threshold for actication
+
+#ifndef PLOOPY_SCROLL_DEBOUNCE
+#    define PLOOPY_SCROLL_DEBOUNCE 5
 #endif
-#ifndef OPT_SCALE
-#    define OPT_SCALE 1  // Multiplier for wheel
+#ifndef PLOOPY_SCROLL_BUTTON_DEBOUNCE
+#    define PLOOPY_SCROLL_BUTTON_DEBOUNCE 100
 #endif
+
 #ifndef PLOOPY_DPI_OPTIONS
 #    define PLOOPY_DPI_OPTIONS \
-        { 1200, 1600, 2400 }
+        { 600, 900, 1200, 1600, 2400 }
 #    ifndef PLOOPY_DPI_DEFAULT
 #        define PLOOPY_DPI_DEFAULT 1
 #    endif
 #endif
 #ifndef PLOOPY_DPI_DEFAULT
 #    define PLOOPY_DPI_DEFAULT 0
 #endif
```

Comment:
It's unclear if this is intentional or not, but `PLOOPY_DPI_DEFAULT` is repeated here twice 

I don't think it has any ill effects, since `PLOOPY_DPI_DEFAULT 1` will always be defined, or the keymap config.h will win, but that does mean that it appears `PLOOPY_DPI_DEFAULT 0` may not be doing anything?

Reply:
if the user specifies a dpi options array, the default isn't set.  So to ensure that it has a default set.